client/js/loading-slow-alert.js

```diff
@@ -19,3 +19,18 @@ setTimeout(function() {
 document.getElementById("loading-slow-reload").addEventListener("click", function() {
 	location.reload();
 });
+
+window.g_LoungeErrorHandler = function LoungeErrorHandler(error) {
+	var title = document.getElementById("loading-title");
+	title.textContent = "An error has occured";
+
+	title = document.getElementById("loading-page-message");
+	title.textContent = "An error has occured that prevented the client from loading correctly.";
+
+	var element = document.createElement("p");
+	element.contentEditable = true;
+	element.textContent = error instanceof ErrorEvent ? error.message : error;
```

Comment:
should be monospace imo

Reply:
I would actually not display the error as part of the UI. I think we should always provide human-friendly error messages (yes, that requires `if`/`switch` statements we improve over time), and if necessary, open dev console.
99% of the time, it won't be completely random errors, but things we know (network issue, bundle issue, etc.) and therefore we can advise on.

Just checking, if you don't display `error.message`, it still appears in dev console, right?

And yes, I know mobile will not allow this. An alternative could be what Chrome does: default error is very human-readable, and a "More info" displays technical details. We could use `details`/`summary` for a rudimentary solution.